rocAL/rocAL/include/augmentations/color_augmentations/node_color_twist.h

```diff
@@ -24,28 +24,23 @@ THE SOFTWARE.
 #include "node.h"
 #include "parameter_factory.h"
 #include "parameter_vx.h"
-#include "graph.h"
```

Comment:
Why graph.h include is removed? is not required any more?
There is a graph.h existing in node_blur.h as well. If it is not needed anymore, please remove from there as well

Reply:
Required